agendas/2022/2022-01-06.md

```diff
@@ -0,0 +1,88 @@
+<!--
+
+Hello! You're welcome to join our working group meeting and add to the agenda
+by following these few steps and guidelines:
+
+   1. Add your name to the list of attendees (in alphabetical order).
+
+      - To respect meeting size, attendees should be relevant to the agenda.
+        That means we expect most who join the meeting to participate in
+      discussion. If you'd rather just watch, check out our YouTube[1].
+
+      - Please include the organization (or project) you represent, and the
+        location (including country code[2]) you expect to be located in during
+        the meeting.
+
+      - If you're willing to help take notes, add "✏️" after your name
+        (eg. Ada Lovelace ✏). This is hugely helpful!
+
+   2. If relevant, add your topic to the agenda (sorted by expected time).
+
+      - Every agenda item has four parts: 1) the topic, 2) an expected time
+        constraint, 3) who's leading the discussion, and 4) a list of any
+        relevant links (RFC docs, issues, PRs, presentations, etc). Follow the
+        format of existing agenda items.
```

Comment:
Following yesterdays's discussion, perhaps add something like: 

> - If your topic is a proposal and it's your first time presenting it, it is likely an RFC0/Strawman proposal. The barrier to entry for an RFC0 to an agenda is intentionally low, writing a few sentences about the problem you're trying to solve and the rough shape of your proposed solution is normally sufficient. It's a good idea to know what you want to get out of the meeting - what feedback do you need? What questions do you need answered? You can file this:
>   - As an issue against the graphql-wg repo
>   - As an RFC document into [the `rfcs/` folder of the graphql-wg repo](https://github.com/graphql/graphql-wg/tree/main/rfcs) - please send a PR
>   - As a [GitHub discussion in the graphql-wg repo](https://github.com/graphql/graphql-wg/discussions)

Maybe a little wordy, it could be stripped down if we can link to this information elsewhere?

Reply:
I like this suggestion. I realized we don't have a direct link to CONTRIBUTING anywhere in here or elsewhere in WG, so I may also add alongside